core/schema.rs

```diff
@@ -508,11 +512,11 @@ pub enum Affinity {
     Numeric,
 }
 
-pub const SQLITE_AFF_TEXT: char = 'a';
-pub const SQLITE_AFF_NONE: char = 'b'; // Historically called NONE, but it's the same as BLOB
-pub const SQLITE_AFF_NUMERIC: char = 'c';
-pub const SQLITE_AFF_INTEGER: char = 'd';
-pub const SQLITE_AFF_REAL: char = 'e';
+pub const SQLITE_AFF_NONE: char = 'A'; // Historically called NONE, but it's the same as BLOB
```

Comment:
do we actually need these or is the Affinity enum enough?

Reply:
Thanks for the review! SQLite uses them to display at query plan, can make them Display trait, or we can figure out better way to display affinity, what do you think?